README.md

```diff
@@ -2,102 +2,33 @@
 [![Build Status](https://drone.srcd.host/api/badges/src-d/landing/status.svg)](https://drone.srcd.host/src-d/landing)
 [![Docker Repository on Quay](https://quay.io/repository/srcd/blog/status "Docker Repository on Quay")](https://quay.io/repository/srcd/blog)
 
-This is our [blog](https://blog.sourced.tech) source. It uses fancy technologies, such as:
-- [Hugo](http://gohugo.io/)
-- [Caddy](https://caddyserver.com/)
+This is our [blog](https://blog.sourced.tech) source.
 
-## Available shortcodes
+# Requirements
 
-### Code scroll
+You need to have [Yarn installed](https://yarnpkg.com/en/docs/install) to build the js and css assets. It also lets you locally serve the blog to test and validate it while you are developing.
 
-If your code is too long you can make it have a fixed height and be scrollable.
-
-```
-{{% codescroll height="500" %}}
-\`\`\`js
-my fancy code
-\`\`\`
-{{% /codescroll %}}
-```
-
-### Center
-
-Center the text
-
-```
-{{% center %}}
-some text
-{{% /center %}}
-```
-
-### Caption
-
-Renders an image with caption
-
-```
-{{% caption src="url to image" title="title of image" %}}
-caption of the image, html and md allowed
-{{% /caption %}}
+You can ensure if it is available in your machine running:
+```shell
+yarn --version; # prints your Yarn version
 ```
 
-### YouTube video
+The blog also uses:
+- [Hugo](http://gohugo.io): to build the static html from the markdown content files; it is automatically downloaded and installed at building time.
+- [Caddy](https://caddyserver.com): to serve the static files in production; it is provided by the caddy docker image
 
-Display a YouTube video
+# Build
 
-```
-{{% youtube VIDEO_ID %}}
-```
-
-### Gist
-
-Display a Gist (even an IPython Notebook)
+You need to satisfy the [project requirements](#requirements), and then to run:
```

Comment:
You have to run that the project root dir, right?

Also we must make clearer what does it mean/happens when you build it.